airflow/jobs/scheduler_job.py

```diff
@@ -1434,6 +1438,23 @@ def _execute_helper(self):
 
         settings.Session.remove()
 
+    def _find_dags_to_process(self, dags: List[DAG], paused_dag_ids: List[str]):
```

Comment:
Now I see the annotation, this should be:
```suggestion
    def _find_dags_to_process(self, dags: List[DAG], paused_dag_ids: Set[str]):
```

Reply:
I can also pick this up in a sepearate PR.

Reply:
Changed it to `Set`.

Reply:
👍